### ISSUE TITLE
Unable to generate device address at hardware-wallet-js start up.

### ISSUE THREAD

**Issue by stdevAlDen** [skycoin/hardware-wallet-js#36]
Running `npm start` give me this err msg in the terminal emulator:

```
> hardware-wallet-js@1.0.0 start /mnt/WORK/SkyCoin/hardware-wallet-js
> node main.js

dev 313a / 1 : usage_page:18d8, usage:29d
[ { vendorId: 12602,
    productId: 1,
    path: '/dev/hidraw0',
    serialNumber: 'E708CBEF2D9DBABD4E7F8494',
    manufacturer: 'SkycoinFoundation',
    product: 'SKYCOIN',
    release: 256,
    interface: 0,
    usagePage: 6360,
    usage: 669 } ]
[Function: DeviceEventObserver]
dev 313a / 1 : usage_page:0, usage:0
dev 313a / 1 : usage_page:2e31, usage:2f30
dev 313a / 1 : usage_page:2e31, usage:2f30
dev 313a / 1 : usage_page:0, usage:0
Skycoin hardware is plugged in
Device Open
dev 313a / 1 : usage_page:0, usage:0
Writing a buffer of length  64 to the device
undefined
Received data of length:  64
That message says its size is:  20
Received header  msg this.kind:  MessageType_Failure  size:  20 buffer lenght:  64 
Remaining bytesToGet: -35
Device Close
Addresses generation received message kind:  MessageType_Failure
Failure message code 13 message:  Mnemonic not set
Promise rejected Error: Mnemonic not set
    at addressGenHandler (/mnt/WORK/SkyCoin/hardware-wallet-js/device-wallet.js:740:24)
    at devReadCallback (/mnt/WORK/SkyCoin/hardware-wallet-js/device-wallet.js:673:9)
    at BufferReceiver.receiveBuffer (/mnt/WORK/SkyCoin/hardware-wallet-js/buffer-receiver.js:46:13)
    at devHandleCallback (/mnt/WORK/SkyCoin/hardware-wallet-js/device-wallet.js:125:36)
```

PD:
Firmware version is Firmware Version 1.6.1

hardware-wallet-go deviceFeatures
2019/01/26 12:39:14 Vendor: Skycoin Foundation
MajorVersion: 1
MinorVersion: 6
PatchVersion: 1
BootloaderMode: false
DeviceId: 453730384342454632443944424142443445374638343934
PinProtection: false
PassphraseProtection: false
Language: 
Label: 
Initialized: false
BootloaderHash: fd5e725b055c94ab5ba73656fccd427ae3714194194efe626ba3883a256ee42f
PinCached: false
PassphraseCached: false
FirmwarePresent: false
NeedsBackup: false
Model: 1
FwMajor: 0
FwMinor: 0
FwPatch: 0
FwVendor: 
FwVendorKeys: 
UnfinishedBackup: false

**Comment by stdevAlDen** [comment]
Solved generating a seed first:
`hardware-wallet-go deviceGenerateMnemonic`
closes #36

**Comment by olemis** [comment]
mnemonimic = seed
